### DISCUSSION TITLE
I cannot login to swagger-stats if there more than one pod active

### DISCUSSION THREAD

**Discussion by zhenisduissekov** [slanatech/swagger-stats#253] [Q&A]
having swagger-stats on two or more pods throws 403(Forbidden) error immediately after login. How can I fix it?

**Comment by NathanSavageKaimai** [comment]
swagger stats isnt aware of any other parallel instances. It sounds to me like your requests are being load balanced when you are only logged into one pod. Look into setting up customAuth in your config. Looking at /lib/swsAuth.js (line 50) and /lib/swssettings.js (line 80), you can write a function that will take the request object and needs to return true if authenticated or false if not. Using this, you can make some kind of common token system so that your pods are aware of each others logins using redis, a db etc. As opposed to 'onAuthenticate', it looks like 'customAuth' will run against every request.
